packages/core/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, Host, Prop, h } from '@stencil/core'
+import { IconProps } from '../../icons'
+
+@Component({
+  tag: 'atom-tag',
+  styleUrl: 'tag.scss',
+  shadow: true,
+})
+export class AtomTag {
+  @Prop() color: 'success' | 'danger' | 'warning' | 'dark' | 'light' | 'neutral'
+  @Prop() icon?: IconProps
+  @Prop() customBackgroundColor: string
+  @Prop() customTextColor: string
```

Comment:
**Question**🤔:  As far as i undertstood these two are not required, i believe a `?` is missing here

Reply:
@schirrel forgot that, thanks! https://github.com/juntossomosmais/atomium/pull/261/commits/a87968cc4865dd8f8f2b07af224a62c8d830783d